src/worker.cpp

```diff
@@ -1185,4 +1183,277 @@ void Worker::collectFrusFromJson()
         }
     }
 }
+
+types::BackupMapVariant Worker::getBackupMap()
+{
+    if (m_parsedJson.contains("backupAndRestore"))
+    {
+        const std::string& l_backupMap =
+            m_parsedJson["backupAndRestore"].value("BackupMap", "");
+        if (constants::VpdBackupMap.find(l_backupMap) !=
+            constants::VpdBackupMap.end())
+        {
+            return constants::VpdBackupMap.at(l_backupMap);
+        }
+    }
+    return std::monostate{};
+}
+
+bool Worker::isBackupAndRestoreSourceOnCache(const std::string& i_pathName)
```

Comment:
I don't see necessity of this API. Please let me know if I am missing something here.

Reply:
Removed this API.